tests/Load/config.sh

```
@@ -0,0 +1,33 @@
+
+prompt_for_input() {
+  local prompt_message=$1
+  local default_value=$2
+  local user_input
+
+  read -r -p "$prompt_message [$default_value]: " user_input
+  echo "${user_input:-$default_value}"
+}
+
+export AWS_PAGER=""
+export REGION=$(prompt_for_input "Enter AWS Region" "us-east-1")
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)